iron-collapse.html

```diff
@@ -105,12 +105,23 @@
         value: false,
         notify: true,
         observer: '_openedChanged'
-      }
+      },
+
+      /**
+       * Set animated to false to disable animations
+       *
+       * @attribute animated
+       */
+      animated: {
+        type: Boolean,
+        value: true
```

Comment:
Since this is the default case, and always there, would it make more sense for this to be `no-animation`, and only be present in those rare cases? Like, when i write
```
<iron-collapse></iron-collapse>
```
I don't assume `animated` is implicitly true because I didn't write it